pkg/operator/ceph/csi/spec.go

```diff
@@ -461,6 +471,54 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 	pluginTolerations := getToleration(r.opConfig.Parameters, pluginTolerationsEnv, []corev1.Toleration{})
 	pluginNodeAffinity := getNodeAffinity(r.opConfig.Parameters, pluginNodeAffinityEnv, &corev1.NodeAffinity{})
 
+	if multusApplied && !v.SupportsMultus() {
+		logger.Warningf("multus is applied but the csi version %q does not support it, need at least %q", v.String(), multusSupportedVersion.String())
```

Comment:
should we not fail the reconcile?

Reply:
I was hesitant too, technically speaking it's going to work but won't survive upgrades. So it's probably better to fail instead, to play it safe.